ProcessMaker/Http/Controllers/Auth/LoginController.php

```diff
@@ -55,6 +58,25 @@ public function showLoginForm()
     {
         $manager = App::make(LoginManager::class);
         $addons = $manager->list();
+        $default = Setting::where('key', 'sso.default.login')->first();
+
+        // Redirectind to default login
+        if (!empty($default) && !empty($addons->toArray())) {
+            $position = $default->getAttribute('config'); // Type int
+            $data = head($addons->toArray())->data;
```

Comment:
$addons->toArray() is converted twice
please validate if **data** property exist